groupDevDependencies.json

```diff
@@ -11,8 +11,8 @@
       ],
       "groupName": "devDependencies (non-major)",
       "automerge": true,
-      "extends": [
-        "schedule:earlyMondays"
+      "schedule": [
+        "before 3am on Monday"
```

Comment:
since you are on it can we switch to Tuesday as some folks are off Monday...
(We discussed it a while ago the idea is to get a more even spread)